.pipelines/containerSourceData/busybox/Dockerfile-Busybox

```diff
@@ -33,8 +33,7 @@ RUN mkdir /staging \
     && pushd /staging \
     && rm -rf boot media mnt opt run \
     && rm -rf usr/lib/sysimage \
-    && rm -rf var/cache \
-    && rm -rf var/lib/rpm; \
+    && rm -rf var/cache; \
```

Comment:
The database can be several MBs large. Is that OK in terms of image size bloat?

Reply:
the size increase is only ~1MB.  Talked with Jon and Mandeep and they are Ok with making db linger in our busybox container.